File: chrisAss/Nokia3310.java
import java.util.Scanner;

public class Nokia3310{

public static void main(String[] args){
Scanner scanner = new Scanner(System.in);
System.out.println("Enter 1 to 13 to Access Menu Options");
System.out.print("1. Phone book\n2. Messages\n3. Chat\n4. Call register\n5. Tones\n6. Settings\n7. Call divert\n8. Games\n9. Calculator\n10. Reminders\n11. Clock\n12. Profiles\n13. Sim services\n");

int menu = scanner.nextInt();
switch(menu){
case 1:
System.out.println("Phone book");
System.out.print("1. Search\n2. Service Nos.\n3. Add name\n4. Erase\n5. Edit\n6.  Assign tone\n7. Send b'card\n8. Options\n9. Speed dials\n10. Voice tags");
System.out.println("Enter 1 to 10 to Access Phone book");
int phoneBook = scanner.nextInt();
switch(phoneBook){
case 1: System.out.println("Search");
break;
case 2: System.out.println("Service Nos.");
break;
case 3: System.out.println("Add name");
break;
case 4: System.out.println("Erase");
break;
case 5: System.out.println("Edit");
break;
case 6: System.out.println("Assign tone");
break;
case 7: System.out.println("Send b'card");
break;
case 8: System.out.println("Options");
		System.out.println("1. Type of view\n2. Memory status");
		System.out.println("Enter 1 or 2 to access Options");
		int options = scanner.nextInt();
		switch(options){
			case 1: System.out.println("Type of view");
			break;
			case 2: System.out.println("Memory status");
			break;
}
			
}
break;
case 9: System.out.println("Speed dials");
break;
case 10: System.out.println("voice tags");
break;
}
case 2:
System.out.println("Messages");
System.out.print("1. Write messages\n2. Inbox\n3. Outbox\n4. Picture messages\n5. Templates\n6. Smileys\n7. Message settings\n8. Info service\n9. Voice mailbox number\n10. Service command editor ");
System.out.println("Input numbers from 1 to 10 to access the message menu\n");
int messageMenu = scanner.nextInt();
switch(messageMenu){
case 1: System.out.println("Write messages");
break;
case 2: System.out.println("Inbox");
break;
case 3: System.out.println("Outbox");
break;
case 4: System.out.println("Picture message");
break;
case 5: System.out.println("Templates");
break;
case 6: System.out.println("Smileys");
break;
case 7: System.out.print("Message settings");
break;
case 8: System.out.println("Info service");
break;
case 9: System.out.println("Voice mailbox number");
break;
case 10: System.out.println("Service command editor\n");
break;
}
case 3:
System.out.println("Chat");
int chatMenu = scanner.nextInt();
switch(chatMenu){
}
case 4:
System.out.println("Call register");
int callMenu = scanner.nextInt();
System.out.println("1. Missed calls\n2. Received calls\n3. Dialled numbers\n4. Erase recent call lists\n5. Show call duration");
switch(callMenu){
case 1: 
System.out.println("Missed calls");
break;
case 2: System.out.println("Received calls");
break;
case 3: System.out.println("Dialled numbers");
break;
case 4: System.out.print("Show call duration");
break;
}
case 5:
System.out.println("Tones");
int callTones = scanner.nextInt();
System.out.println("1. Ringing tone\n2. ringing volume\n3. Incoming call alert\n4. Composer\n5. Message alert tone\n6. Keypad tones\n7. Warning and game tones\n8. Vibrating alert\n9. Screen saver");
switch(callTones){
case 1: System.out.println("Ringing tone");
case 2: System.out.println("Ringing volume");
case 3: System.out.println("Incoming call alert");
case 4: System.out.println("Composer");
case 5: System.out.println("Message alert tone");
case 6: System.out.println("Warning and game tones");
case 7: System.out.print("Warning and game tones");
case 8: System.out.println("Vibrating alert");
case 9: System.out.println("Screen saver");
}
case 6:
System.out.println("Settings");
int callSettings = scanner.nextInt();
System.out.println("1. Call settings\n2. Phone settings\n3. Security settings\n4. Restore factory settings");
switch(callSettings){
case 1: System.out.println("Call settings");
case 2: System.out.println("Phone settings");
case 3: System.out.println("security settings");
case 4: System.out.println("Restore factory settings");
}
case 7:
System.out.println("Call divert");
int calllMenu = scanner.nextInt();
switch(calllMenu){
}
case 8:
System.out.println("Games");
int gameMenu = scanner.nextInt();
switch(gameMenu){
}
case 9:
System.out.println("Calculator");
int calMenu = scanner.nextInt();
switch(calMenu){
}
case 10:
System.out.println("Reminders");
int remMenu = scanner.nextInt();
switch(remMenu){
}
case 11:
System.out.println("Clock");
int cloMenu = scanner.nextInt();
switch(cloMenu){
}
case 12:
System.out.println("Profiles");
int proMenu = scanner.nextInt();
switch(proMenu){
}
case 13:
System.out.println("SIM services");
int simMenu = scanner.nextInt();
switch(simMenu){
}
}
}
}

File: GamesOfJava/Nokia3310.py
print("List of menu functions")
print("1.  Phone book")
print("2.  Messages")
print("3.  Chat")
print("4.  Call register")
print("5.  Tones")
print("6.  Settings")
print("7.  Call divert")
print("8.  Games")
print("9.  Calculator")
print("10. Remainders")
print("11. clock")
print("12. Profiles")
print("13. SIM services")



functions = int(input("Enter number 1 to 13 to access the menu : "))
while functions >= 1 or functions <= 13:  
	match functions:
		case 1: 
			print("1. Search")
			print("2. Service Nos.")
			print("3. Add name")
			print("4. Erase")
			print("5. Edit")
			print("6. Assign tone")
			print("7. Send b'card")
			print("8. Options")
			print("9. Speed dials")
			print("10.Voice tags")
			
			phonebook1 = int(input("select from options 1-10 :"))
			match phonebook1:
				case 1: 
					print("Search")
				case 2: 
					print("Service Nos.")
				case 3:
					print("Add name")
				case 4:
					print("Erase")
				case 5:
					print("Edit")
				case 6: 
					print("Assign tone")
				case 7: 
					print("Send b'card")
				case 8: 
					print("Options")
				case 9:
					print("Speed dials")
				case 10:
					print("Voice Tags")
				
		
				
		case 2:
			print("Messages")
			print("1. Write messages")
			print("2. Inbox")
			print("3. Outbox")
			print("4. Picture Messages")
			print("5. Templates")
			print("6. Smileys")
			print("7. Message settings")
			print("9. Info service")
			print("10.Voice mailbox number")
			print("11.Service command editor")


			Message1 = int(input("Select from options 1 t0 10 : "))
			match message1: 
				case 1:
					print("Write messages")
				case 2: 
					print("Inbox")
				case 3:
					print("Outbox")
				case 4:
					print("Picture messages")
				case 5:
					print("Templates")
				case 6:
					print("Smileys")
				case 7:
					print("Message settings\n1.Set 1\n2. Common")
			
		case 3:
			print("Chat")
				

		case 4:
			
			print("Call register")
			print("Missed calls")
			print("Received calls")
			print("Dialled numbers")
			print("Erase recent call lists")
			print("Show call duration")

			call_register_one = int(input("select from options 1 to 6 to access call register : "))
			match call_register_one:
				case 1:
					print("Call register")
				case 2:
					print("Missed calls")
				case 3:
					print("Received calls")
				case 4: 
					print("Dialled numbers")
				case 5:
					print("Erase recent call lists")
				case 6:
					print("Show call duration")
						


				
		case 5:
			print("Tones")
			print("1. Ringing tone")
			print("2. Ringing volume")
			print("3. Incoming call alert")
			print("4. Composer")
			print("5. Message alert tone")
			print("6. Keypad tones")
			print("7. Warning and Game tones")
			print("8. Vibrating alert")
			print("9. Screen saver")


			nokia_tones = int(input("select from options 1 to 9 to access tones : "))
			match nokia_tones:		
				case 1:
					print("Ringing tone")
				case 2:
					print("Ringing volume")
				case 3:
					print("Incoming call alert")
				case 4:	
					print("Composer")
				case 5:
					print("Message alert tone")
				case 6: 
					print("Keypad tones")
				case 7:
					print("Warning and Game tones")
				case 8:
					print("Vibrating alert")
				case 9:
					print("Screen saver")
				
		case 6:
			print("Settings")
			print("Call settings")
			print("Phone settings")
			print("Security settings")
			print("Restore factory settings")



			nokia_settings = int(input("select from options 1 to 4 to access"))
				
			match nokia_settings:
				case 1:
					print("Call settings")
					print("1. Automatic redial")
					print("2. Speed dialling")
					print("3. Call waiting options")
					print("4. Own numbers sending")
					print("5. Phone line in use")
					print("6. Automatic answer")
					nokia_call_settings = int(input("Select from options 1 to 6 to access call settings"))
					
					match nokia_call_settings:
						case 1:
							print("Automatic redial")
						case 2:
							print("Speed dialling")
						case 3:
							print("Call waiting options")
						case 4:
							print("Own numbers sending")
						case 5:
							print("Phone line in use")
						case 6:
							print("Automatic answer")
			
				case 2:
					print("Phone settings")
					print("1. Language")
					print("2. Cell info display")
					print("3. Welcome note")
					print("4. Network selection")
					print("5. lights")
					print("6. Confirm SIM service actions")
				
					nokia_print_options = int(input("Select from options 1 to 6 to access phone settings"))

					match nokia_print_options:
						case 1: 
							print("Language")
						case 2:
							print("Cell info display")
						case 3:
							print("Welcome note")
						case 4:
							print("Network selection")
						case 5:
							print("lights")
						case 6:
							print("Confirm SIM service actions")

				case 3:
					print("Security settings")
					print("1. PIN code request")
					print("2. Call barring service")
					print("3. Fixed Dialling")
					print("4. Closed user group")
					print("5. Phone security")
				case 4:
					print("Restore factory settings")
				
				
		case 7: 
			print("Call divert")
		case 8:
			print("Games")
		case 9: 
			print("Reminders")
		case 10:
			print("Calculator")
		case 11: 
			print("Clock")
		case 12:
			print("Profiles")
		case 13:
			print("SIM services")

	
	
